src/Tribe/Linked_Posts.php

```diff
@@ -1264,7 +1264,9 @@ class="tribe-dropdown linked-post-dropdown hide-before-select2-init"
 				data-force-search
 				<?php endif; ?>
 			>
-				<option></option>
+				<option value="-1" <?php selected( empty( $current ) ); ?>>
+					<?php echo esc_html( $label ); ?>
+				</option>
```

Comment:
This is required so that the VenueID is populated with a non-blank value, which allows venues to be created.